js/runner/Dockerfile

```diff
@@ -9,21 +9,35 @@ ARG STDLIB_IMAGE=reachsh/stdlib:$REACH_VERSION
 FROM $JS_DEPS_IMAGE as js-deps
 
 # hadolint ignore=DL3006
-FROM $NODE_IMAGE as app0
+# FROM $NODE_IMAGE as app0
 
-WORKDIR /app
-COPY --from=js-deps /js-deps/node_modules/ /app/node_modules/
```

Comment:
This basically throws the whole idea of `js-deps` in the trash, which makes me sad.